Modules/_datetimemodule.c

```diff
@@ -2905,7 +2905,7 @@ date_today(PyObject *cls, PyObject *dummy)
 @classmethod
 datetime.date.fromtimestamp
 
-    timestamp: object
+    timestamp: 'O'
```

Comment:
Why this change?

Reply:
That was an accident, I was playing around with the argument clinic spec to try to get it to generate a `METH_VARARGS` method. Obviously it didn't work, so I'll revert it.